### DISCUSSION TITLE
table export limit

### DISCUSSION THREAD

**Discussion by gpjorgenson** [Budibase/budibase#4989] [Help] [answered]
When I try to export an entire table, budibase only exports 5000. I have not been able to find a way to adjust or eliminate that parameter. Can anyone tell me how to do that?

**Comment by mike12345567** [comment]
Hi @gpjorgenson - just to confirm, are you trying to export a SQL table?

**Reply by gpjorgenson** [reply]
Yes, it is a postgresql table.

**Reply by mike12345567** [reply]
Hi @gpjorgenson - I've added a change to PR https://github.com/Budibase/budibase/pull/4991 - once released it'll allow you to change the environment variable `SQL_MAX_ROWS` for the app service to whatever limit you'd desire; this is for self host.

For the Cloud, we have to maintain the limit of 5000 rows per query (this extends to export as well) and therefore this cannot be raised.

**Reply by gpjorgenson** [reply]
Thank you. We are self host with the database on our own server. I will look forward to that release.

**Comment by gpjorgenson** [comment]
Yes

**Comment by PMcLeanGLX** [comment]
Which container in a Docker deployment would I add this env variable to?

**Reply by mike12345567** [reply]
environment variables to do with app operations should be added to the `budibase/apps` containers.

**Reply by PMcLeanGLX** [reply, marked as answer]
When I add SQL_MAX_ROWS to the apps container I get a 502 error (Bad Gateway). Its pulling budibase/apps:latest, but when I deploy using the script and docker-compose, it deploys budibase/apps.

Disregard, just added it to the docker-compose and redeployed.
